app/code/Dev/ProductComments/Controller/Adminhtml/Comment.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Dev\ProductComments\Controller\Adminhtml;
+
+use Magento\Framework\Registry;
+use Magento\Framework\App\Action\Action;
+use Magento\Framework\App\Action\Context;
+use Magento\Backend\Model\View\Result\Page;
+
+abstract class Comment extends Action
```

Comment:
Any specific reason for defining a class as abstract?

Reply:
not that I can think of probably coped form some tutorial article without thinking.  I can fix it if it needs to be fixed.